### PR TITLE
CRW-1184 :: split XP1 devfile into XP1 (s390x + ppc64le) + XP2 (amd64)

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

* CRW-1184 add devfile for XP2 (x86_64 only); rename 00_java-microprofile -> 00_java-microprofile-xp1 (ppc64le and s390x only)
* update to latest registry.redhat.io/jboss-eap-7/eap-xp1-openj9-11-openshift-rhel8:1.0.2-5.1604581793; remove toggle in build/scripts/swap_images.sh

Change-Id: Ic0e114ff06e5373ba88ab89e694f77f8012f0a3c
Signed-off-by: nickboldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/CRW-1184

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: dependencies/che-devfile-registry/devfiles/00_java-microprofile-xp2/devfile.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+apiVersion: 1.0.0
+metadata:
+  generateName: jboss-eap-quickstart-
+projects:
+  -
+    name: jboss-eap-quickstart
+    source:
+      type: git
+      location: "https://github.com/crw-samples/microprofile-quickstart.git"
+      branch: "xp-2.0.x"
```
Comment (nickboldt) [inline dependencies/che-devfile-registry/devfiles/00_java-microprofile-xp2/devfile.yaml:11]: @svor do we need anything other than the branch here to make this devfile work for XP2? I only changed 2 lines from the XP1 version -- branch and sidecar image.

Comment (svor) [reply dependencies/che-devfile-registry/devfiles/00_java-microprofile-xp2/devfile.yaml:11]: that's enough

**Review by svor — APPROVED** [review]
LGTM

File: dependencies/che-devfile-registry/devfiles/00_java-microprofile-xp2/devfile.yaml (hunk 1 above)
Comment (svor) [reply dependencies/che-devfile-registry/devfiles/00_java-microprofile-xp2/devfile.yaml:11]: that's enough!

**Pull request merged**